buildlib/pr/distro.yml

```
@@ -113,3 +118,49 @@ jobs:
           UCX_V11_LIB_PATH: $(System.DefaultWorkingDirectory)/ucx-1.11/install/lib
           UCX_PR_LIB_PATH: $(System.DefaultWorkingDirectory)/ucx_bin_$(Build.BuildId)/install/lib
         displayName: Test wire compatibility server on master client v1.11
+
+  - job: Test_compatibility
+    dependsOn: Distros
+    displayName: Test UCP_HELLO_WORLD
+    workspace:
+      clean: all
+    pool:
+      name: MLNX
+      demands:
+      - ucx_docker -equals yes
+      - ucx_amd -equals no
+    strategy:
+      matrix:
+        centos7_ib:
+          CONTAINER: centos7_ib
+    steps:
+      - checkout: none
+        clean: true
+      - task: DownloadBuildArtifacts@0
+        displayName: 'Download Build Artifacts'
+        inputs:
+          artifactName: ucx_bin_$(Build.BuildId)
+          downloadPath: $(System.DefaultWorkingDirectory)
+      - bash: chmod u+rwx $(System.DefaultWorkingDirectory)/ucx_bin_$(Build.BuildId) -R
+      - bash: |
+          set -Eex
+          git clone --depth 1 -b v1.11.x https://github.com/openucx/ucx.git $(System.DefaultWorkingDirectory)/ucx-1.11
+          cd $(System.DefaultWorkingDirectory)/ucx-1.11
+          ./autogen.sh
+          ./contrib/configure-release --prefix=$(System.DefaultWorkingDirectory)/ucx-1.11/install --enable-examples
+          make -s -j `nproc`
+          make install -s -j `nproc`
+        displayName: Build ucx-1.11 version
+
+      - bash: |
+          set -Eex
+          ldd $(System.DefaultWorkingDirectory)/ucx_bin_$(Build.BuildId)/install/examples/hworld
+          UCX_LOG_LEVEL=info LD_LIBRARY_PATH=$UCX_PR_LIB_PATH/ $(System.DefaultWorkingDirectory)/ucx_bin_$(Build.BuildId)/install/examples/hworld &
```

```suggestion
          UCX_LOG_LEVEL=info UCX_TLS=^sm LD_LIBRARY_PATH=$UCX_PR_LIB_PATH/ $(System.DefaultWorkingDirectory)/ucx_bin_$(Build.BuildId)/install/examples/hworld &
```
because there are issues in wire compatibility over shared memory
@yosefe what do you think?

---

ok, let's do this to make the test pass